mod_reforged/hooks/config/strings.nut

```diff
@@ -310,12 +310,14 @@ local vanillaDescriptions = [
 		ID = "perk.anticipation",
 		Key = "Anticipation",
 		Description = ::UPD.getDescription({
-			Fluff = "I saw these coming!",
+			Fluff = "I saw that coming!",
 			Effects = [{
 				Type = ::UPD.EffectType.Passive,
 				Description = [
-					"Take reduced damage from the first " + ::MSU.Text.colorPositive(2) + " attacks you receive each battle.",
-					"This reduction is a percentage equal to your current [Ranged Defense|Concept.RangeDefense] plus an additional " + ::MSU.Text.colorPositive("10%") + " for each tile between the attacker and you."
+					"Gain " + ::MSU.Text.colorPositive("+10") + " [Ranged Defense.|Concept.RangeDefense]",
```

Comment:
This part of the perk seems to not be implemented yet.
When you do that, don't forget to gate it behind a member variable so it can be easier turned off or changed.